### ISSUE TITLE
Support multiple images as input

### ISSUE THREAD

**Issue by AntonioND** [Optiroc/SuperFamiconv#45]
grit accepts providing multiple images as input and creating a combined palette or tileset as output.

SuperFamiconv lets you do things like provide a png that acts as palette or tileset and only generate a map from the other png file with the full image. This is very useful, but not the same thing as what grit does. You need to have created the combined palette or tileset beforehand.

It would be great if SuperFamiconv could support the first behaviour.

**Comment by H-A-M-G-E-R** [comment]
As a workaround, you can combine multiple images to a single image.

**Comment by AntonioND** [comment]
Yes, I ended up doing that with a tool I created just for my game https://github.com/AntonioND/ucity-advance/tree/85a97c12587530e67b69595e0a360f10adc6fb41/tools/pngs2strip but it's a bit awkward to do it.

However, I understand that coming up with a good way to input multiple images into the tool would be tricky.

**Comment by Optiroc** [comment]
Done in 5be8d79e14c61488fb751ffef2af87604387377f
